src/configSchema.ts

```diff
@@ -15,13 +15,13 @@ const ZodErrorMessages = {
 	fuzzySizeThreshold: "fuzzySizeThreshold must be between 0 and 1.",
 	injectUrl:
 		"You need to specify rtorrentRpcUrl, transmissionRpcUrl, qbittorrentUrl, or delugeRpcUrl when using 'inject'",
-	recheckWarn:
-		"It is strongly recommended to not skip rechecking for risky or partial matching mode.",
-	windowsPath: `Your path is not formatted properly for Windows. Please use "\\\\" or "/" for directory separators.`,
-	qBitFlatLinking:
-		"Using Automatic Torrent Management in qBittorrent without flatLinking enabled can result in injection path failures.",
+	//TODO for some reason this is not tabbed right? needs \t to align. - I think it's because its refined outside of a primitive
+	// I KNOW YOU HATE THIS IM SORRY MG :(
+	windowsPath: `\t\t\tYour path is not formatted properly for Windows. \n\t\t\t\tPlease use "\\\\" or "/" for directory separators.`,
```

Comment:
what does "this is not tabbed right" mean though 🤨

as far as I'm concerned, "tabbed right" means left-aligned

Reply:
ok, just delete the commentary and I'll make my peace with the tabs.